Snippets/Snippets_6/Snippets_6.csproj

```diff
@@ -21,6 +21,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <NoWarn>1998</NoWarn>
```

Comment:
@andreasohlund nope. this is unique to snippets where we r not doing any async actions